packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -60,7 +70,7 @@ const MenuInspectorControls = ( {
 					/>
 					{ isOffCanvasNavigationEditorEnabled ? (
 						<OffCanvasEditor
-							blocks={ innerBlocks }
+							blocks={ clientIdsTree }
```

Comment:
This is the change the broke the edit button. Could it break other things that are relying on `blocks` to be an array of block objects rather than just client ids?

Reply:
This is a pretty significant change. Why is it necessary?

Also this will likely diverge us from the primary `<ListView>` which presumably accepts `blocks`?

Reply:
It's the other way around, the main List View uses a clientIdsTree. There's a bit more detail in the PR description.

I do think the `blocks` prop might be confusing, but the root cause is really that the `innerBlocks` data doesn't include controlled blocks.

Reply:
> It's the other way around, the main List View uses a clientIdsTree. There's a bit more detail in the PR description

🤦 Apologies

Reply:
@talldan We extracted this to its own PR in https://github.com/WordPress/gutenberg/pull/46541